packages/widgetbook/lib/src/knobs/color_knob.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:widgetbook/src/knobs/knobs.dart';
+
+const _kFallbackColor = 0xFFFF0000;
+
+extension on String {
+  Color asColor() {
+    if (length != 7) {
+      return const Color(_kFallbackColor);
+    }
+    return Color(int.tryParse(replaceFirst('#', '0xFF')) ?? _kFallbackColor);
+  }
+}
```

Comment:
Ok for now, but I think having a fallback color is probably a bit weird from a UX perspective.